www/js/share/services/account.service.js

```diff
@@ -109,21 +109,28 @@ angular.module("binary").service("accountService", function(websocketService, ap
 			 */
     this.add = function(_account) {
         const account = {
-            id        : _account.loginid,
-            token     : _account.token,
-            currency  : _account.currency,
-            email     : _account.email,
-            country   : _account.country,
-            is_default: false
+            id                  : _account.loginid,
+            token               : _account.token,
+            currency            : _account.currency,
+            email               : _account.email,
+            country             : _account.country,
+            is_disabled         : _account.is_disabled,
+            is_ico_only         : _account.is_ico_only,
+            is_virtual          : _account.is_virtual,
+            excluded_until      : _account.excluded_until,
+            landing_company_name: _account.landing_company_name
         };
 
         const accountList = this.getAll();
-
         if (_.find(accountList, ["id", account.id])) {
-            return;
+            const defaultAccountIndex = _.findIndex(accountList, ["id", account.id]);
```

Comment:
IMO, it's better not to set that the account is default or not and let do it in 'accountService.setDefault' function.